src/sntp.c

```diff
@@ -42,7 +42,7 @@ static void sntp_cb(struct mg_connection *c, int ev, void *ev_data) {
   if (ev == MG_EV_READ) {
     int64_t milliseconds = mg_sntp_parse(c->recv.buf, c->recv.len);
     if (milliseconds > 0) {
-      MG_INFO(("%lu got time: %lld ms from epoch", c->id, milliseconds));
+      MG_DEBUG(("%lu got time: %lld ms from epoch", c->id, milliseconds));
```

Comment:
mmm... I'm not sure...
We have other events that are INFO, in MIP for example. Yes, those are more important...